### DISCUSSION TITLE
tell me please, what method and sendler to catch the change of the group to the supergroup?

### DISCUSSION THREAD

**Discussion by paantya** [eternnoir/pyTelegramBotAPI#1717] [Q&A]
tell me please, what method and sendler to catch the change of the group to the supergroup?

1. What version of pyTelegramBotAPI are you using?
last

2. What OS are you using?
ubuntu18

3. What version of python are you using?
last

**Comment by Badiboy** [comment]
Do not use Issues for questions.

**Comment by coder2020official** [comment]
Create a handler with content types of migrate_to_chat_id or migrate_from_chat_id
#1683
